docs: add remote faq for older linux versions (#6953)
microsoft/vscode-docs: docs/remote/faq.md

### What Linux packages or libraries need to be installed on a host to use Remote Development?

Remote Development requires kernel >= 4.18, glibc >=2.28, and libstdc++ >= 3.4.25. Recent x86_64 glibc-based distributions have the best support, but exact requirements can vary by distribution.

Support for musl-based [Alpine Linux](https://alpinelinux.org) is available for the Dev Containers and WSL extensions and ARMv7l (AArch32) / ARMv8l (AArch64) is available in Remote - SSH. However, native dependencies in certain extensions may cause them not to function on non-x86_64 glibc distributions. Note that experimental ARMv8l (AArch64) is available in [VS Code Insiders](https://code.visualstudio.com/insiders/) only.

See [Remote Development with Linux](/docs/remote/linux.md) for additional details.

### Can I run VS Code Server on older Linux distributions?

> Starting with `1.86` release of VS Code, the minimum toolchain for building the remote server was raised after assessment and the prebuilt servers are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.

Reviewer: ```suggestion
Starting with `1.86` release of VS Code, the minimum requirements for the build toolchain of the remote server were raised. The prebuilt servers distributed by VS Code are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.
```

> Starting with `1.86` release of VS Code, the minimum requirements for the build toolchain of the remote server were raised. The prebuilt servers distributed by VS Code are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.

Reviewer 2: ```suggestion
Starting with VS Code release 1.86, the minimum requirements for the build toolchain of the remote server were raised. The prebuilt servers distributed by VS Code are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.
```

> If your platform does not meet the requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version `1.85` to continue usage of the Remote Development. Following are the ways to dowgrade the VS Code client on desktop and web,

Reviewer: ```suggestion
If your platform does not meet the requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version `1.85` to continue usage of the Remote Development. You can downgrade the VS Code client both on desktop and web:
```

> If your platform does not meet the requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version `1.85` to continue usage of the Remote Development. You can downgrade the VS Code client both on desktop and web:

Reviewer 2: ```suggestion
If your setup does not meet these requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version 1.85 to continue using Remote Development. You can downgrade the VS Code client on both desktop and web:
```

> * For desktop, you can download `1.85` release from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform ensure to pin the package version in apt or exclude it in yum to prevent automatic updates.

Reviewer: ```suggestion
* On desktop, you can download the `1.85` release from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform, make sure to disable updates to stay on that version. A good recommendation is to have 1.85 as a separate installation, set up with [Portable Mode](https://code.visualstudio.com/docs/editor/portable). That way, you won't affect your main desktop VS Code version.
```

> * On desktop, you can download the `1.85` release from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform, make sure to disable updates to stay on that version. A good recommendation is to have 1.85 as a separate installation, set up with [Portable Mode](https://code.visualstudio.com/docs/editor/portable). That way, you won't affect your main desktop VS Code version.

Reviewer 2: ```suggestion
* On desktop, you can download the VS Code release 1.85 from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform, make sure to disable updates to stay on that version. A good recommendation is to have release 1.85 as a separate installation, set up with [Portable Mode](https://code.visualstudio.com/docs/editor/portable). That way, you won't affect your main desktop VS Code version.
```

> * For web, you can add the following query argument [?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to force `1.85` version of the client.

Reviewer: ```suggestion
* On web, you can add the following query argument [?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to use `1.85`.
```

> * On web, you can add the following query argument [?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to use `1.85`.

Reviewer 2: ```suggestion
* On web, you can add the following query argument [`?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2`](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to use VS Code release 1.85.
```
